util/opentelemetry-util-genai/src/opentelemetry/util/genai/environment_variables.py

```diff
@@ -43,3 +43,13 @@
    <https://filesystem-spec.readthedocs.io/en/latest/features.html#url-chaining>`_ for advanced
    use cases.
 """
+
+OTEL_INSTRUMENTATION_GENAI_UPLOAD_FORMAT = (
```

Comment:
I also considered making the existing environment variable support a python format string and checking the file extension insetad e.g.

```shell
OTEL_INSTRUMENTATION_GENAI_UPLOAD_FORMAT="s3://foo/bar/{uuid}_{type}.jsonl"
```

but figured it's maybe too complicated. wdyt?